package.json

```
@@ -23,13 +23,14 @@
     "test:debug": "node --inspect-brk -r tsconfig-paths/register -r ts-node/register node_modules/.bin/jest --runInBand",
     "test:e2e": "npm run test -- --runInBand --config ./test/e2e/jest-e2e.json --forceExit",
     "db": "docker compose up",
-    "db:deploy": "docker compose up -d",
+    "db:start": "docker compose up -d",
+    "db:stop": "docker compose stop",
     "db:remove": "docker compose down -v",
-    "db:redeploy": "npm run db:remove; npm run db:deploy",
-    "db:stop": "docker compose stop ",
+    "db:reset": "npm run db:remove; npm run db:start",
     "db:push-schema": "npx prisma db push",
     "db:migrate": "npx prisma migrate deploy",
-    "postdb:deploy": "sleep 1.5; npm run db:migrate",
+    "db:push-sql-functions": "ts-node script/push-sql-functions.ts",
```

```suggestion
    "db:push-functions": "ts-node script/push-sql-functions.ts",
``` 
`db:` probably provides enough context